### DISCUSSION TITLE
Time tracker with applet?

### DISCUSSION THREAD

**Discussion by hublol** [BuddiesOfBudgie/budgie-desktop#602] [General]
Hello,

I recently switched from Cinnamon to Budgie, mostly due to an annoying defect that has been bugging me for years. So far, I am pretty happy with Budgie.
But there is one feature that I miss a lot: a time tracker applet. I was previously using this one (hamster time tracker):
https://cinnamon-spices.linuxmint.com/applets/view/140

Is there something similar available for Budgie? I need to see on which project I am currently working on in the taskbar, and be able to easily switch from one project to another.

Thanks!

**Comment by Jeffv11** [comment]
I too, am a fan of the old Hamster Time Tracker. Hublol, please let me know if you find a similar applet for Budgie. Thx - Jeffala
